src/eins-hwinfo.c

```diff
@@ -232,7 +233,7 @@ typedef struct _LscpuFieldType {
    * field names, which is presumably a bug caused by re-using the field names
    * from the colon-separated default output.
    */
-  const gchar *names[3];
+  const gchar *names[4];
```

Comment:
Not needed, this array size is the number of different names by which the field can be known, not the length of `LSCPU_FIELDS`